resources/app/login/Login.tsx

```diff
@@ -15,91 +15,101 @@ import {
-import { RootState, ActionCreator } from '../../store/combined';
-import { bindActionCreators } from '@reduxjs/toolkit';
+import { sessionStore } from '@store/sessionStore';
+import Axios from 'axios';
 
-export default function Login(){
-    
-    const dispatch = useDispatch()
-    const [ getUsername,setUsername ] = useState("");
-    const [ getPassword,setPassword ] = useState("");
-    const getToken = useSelector((state: RootState) => state.tokenReducer);
-    const token = bindActionCreators(ActionCreator.setToken,dispatch);
-    const removeToken = bindActionCreators(ActionCreator.removeToken,dispatch);
+interface LoginFormProps {
+
+}
+
+function LoginForm(props: LoginFormProps) {
+    const [getUsername, setUsername] = useState("")
+    const [getPassword, setPassword] = useState("")
+    const setToken = sessionStore((state) => state.setToken)
+    const toast = useToast();
     const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
         e.preventDefault();
-        token("Ini Token")
-        console.log(getToken)
+        Axios.post("/api/public/login", { username: getUsername, password: getPassword, csrf: "" }).then((data) => {
+            setToken("ABCDEFU")
+            toast({ title: "Request status", description: JSON.stringify(data.data), position: "top-right", status: "info" })
+        });
     }
+    return (
+        <form onSubmit={handleSubmit}>
+            <Stack
+                spacing="4"
+                p="1.5rem"
+                boxShadow="md"
+            >
+                <FormControl
+                    size="sm"
+                >
+                    <FormLabel htmlFor="email">Email Address</FormLabel>
+                    <Input type="email" name="email" placeholder="Email" value={getUsername} onChange={(e: any) => setUsername(e.target.value)} required />
+                </FormControl>
+                <FormControl
+                    size="sm"
+                >
+                    <FormLabel htmlFor="password">Password</FormLabel>
+                    <Input type="password" name="password" placeholder="Password" value={getPassword} onChange={(e: any) => setPassword(e.target.value)} required />
+
+                </FormControl>
+                <Button
+                    type="submit"
+                    colorScheme="teal"
+                    size="sm"
+                    variant="solid"
+                >LOGIN</Button>
+                <HStack
+                    flexDir="row"
+                    justifyContent="center"
+                    alignItems="center"
+                    borderTop="1px"
+                    borderColor={useColorModeValue("gray.200", "gray.800")}
+                >
+                    <Link to="/db/manager" style={{ paddingTop: "1rem" }}>
+                        <Text fontSize={11.5} color="teal">Database Manager</Text>
+                    </Link>
+                </HStack>
+            </Stack>
+        </form>
+    );
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/bluespada/go-telescope/pullRequest?prid=9653145)
```suggestion
  );
```